content/Objects/Objects.facts.md

```diff
@@ -0,0 +1,37 @@
+### Facts for Objects lesson:
+
+An object is the actual representation or instance of a class.
+
+The `new` keyword is used to create an object as an instance of a class.
+
+`Instantiation` is the process of creating a new object of a class.
+
+To create an object, write the name of the object followed by an equal sign `=`; the `new` keyword and its corresponding class name. One way to write it is this way:
+
+```php
+$objectName = new className();
+```
+
+The object operator `->` is used to access methods and properties of an object.
```

Comment:
![theMethods_OOP_Objects](https://user-images.githubusercontent.com/38550259/60631603-ce683500-9e32-11e9-898b-f82fd6bcb0d0.jpg)

Kindly check this out. Let me know what you think. Thanks.

Reply:
Same here... as what I have told you. Please make sure to check how it is written in actual before testing. There is no issue found on my end. Please try to check without removing the symbol `->` as part of the statement. Thanks.

![image](https://user-images.githubusercontent.com/25945790/60632700-b3e58a00-9e39-11e9-9a18-b4f678aa15e6.png)